core/os/device/deviceinfo/cc/android/query.cpp

```diff
@@ -40,14 +40,14 @@ typedef uint8_t GLubyte;
 
 namespace {
 
-device::DataTypeLayout* new_dt_layout(int size, int alignment) {
+device::DataTypeLayout *new_dt_layout(int size, int alignment) {
```

Comment:
I think we've been putting the stars next to the type, not the name.

Reply:
Yep, I assumed clang-format/presubmit would have fixed that up for me. I will fix.